product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/01_data_pump_utility.mdx

```diff
@@ -113,4 +113,8 @@ See an important [note](../01_mp_schema_extraction/#about-file-encoding) about t
    ```shell
     impdp <Username>@<ConnectIdentifier> DIRECTORY=DMPDIR TRANSFORM=OID:n,SEGMENT_ATTRIBUTES:n SQLFILE=YourSchemas.sql DUMPFILE=schemas_metadata.dump parfile=import.par
    ```
-See an important [note](../01_mp_schema_extraction/#about-file-encoding) about the file encoding format expected by the Migration Portal.  
+See an important [note](../01_mp_schema_extraction/#about-file-encoding) about the file encoding format expected by the Migration Portal.
+
+## Known Issues and Additional Notes
+
+- Migration Portal might fail to parse your SQL file if you create a database link using the IDENTIFIED BY clause with Oracle's quote operator, for example, `IDENTIFIED BY VALUES q'[:1]'`. To parse your file successfully, try using an actual password, for example, `IDENTIFIED BY my_password`.
```

Comment:
```suggestion
- Migration Portal might fail to parse your SQL file if you create a database link using the IDENTIFIED BY clause with Oracle's quote operator; for example, `IDENTIFIED BY VALUES q'[:1]'`. To parse your file successfully, try using an actual password; for example, `IDENTIFIED BY my_password`.
```
I'm pretty sure that we want to use semicolons but you might want to confirm that with Betsy.

Reply:
Fixed.